src/main/resources/templates/resource2.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+		<title>National Cybersecurity Laboratories - Resource</title>
+		<object th:include="fragments/header :: header" th:remove="tag" />
+	</head>
+	<body class="mybg loginbackground">
+	<div th:replace="fragments/dashboard_navigation :: dashboardnav">&nbsp;</div>
+	<div class="container">
+		<p>&nbsp;</p>
+		<p>&nbsp;</p>
+		<p>&nbsp;</p>
+		<p>&nbsp;</p>
+		<!--here make changes-->
+		<!--<div class="tab-content col-sm-10 dashpanel_admin">-->
+		<div class="row dashpanel col-sm-12">
+		<!--<div class="tab-content col-sm-10">-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)